src/sonic-yang-mgmt/tests/test_sonic_yang_mgmt.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
Curious! Who is the consumer of this skeleton base ?

Reply:
The Python Library for yang(PLY) will be build using this. i.e code 3784 & 3891.
Note: as decided for YANG models will have a separate build and pkg system now from location src/sonic-yang-models. Both CVL and PLY can use it.